docs/Full-page.md

```diff
@@ -21,6 +21,9 @@ public function middleware($middlewareQueue) {
             'when' => function (ServerRequest $request, Response $response) {
                 return $request->is('get');
             },
+            'keyGenerator' => function ($key) {
+                return $key; 
```

Comment:
This must be kept empty. This kind of default is dangerous example - you had that also in your initial PR
It would lead to possible OS security issues as anyone passing a foldered path structure could potentially access the system files using ../ hack etc.
So it would be best to keep slugging example here too or just add `// Implement`

Reply:
Ok. I think I will us the way the 4.x branch uses the keyGenerator. As in; if the key genereator exists use only that and ignore everything else (such as prefixes).